xml/operations-compute-using_flavor_metadata.xml

```diff
@@ -8,154 +8,42 @@
 <section xmlns="http://docbook.org/ns/docbook" xmlns:xi="http://www.w3.org/2001/XInclude" xmlns:xlink="http://www.w3.org/1999/xlink" xml:id="topic_vhs_12v_vw">
  <title>Using Flavor Metadata to Specify CPU Model</title>
  <para>
-  In &kw-hos; Nova, the <literal>ComputeCapabilitiesFilter</literal>
-  scheduling filter matches the exact CPU model of the compute host and the
-  requested compute instance model. It doesn't entertain the libvirt's
-  capability of emulating the hosts CPU model to its guests if the guest's CPU
-  model is older or equal to that of the hosts.
+  In &kw-hos; &o_comp;, the ComputeCapabilitiesFilter checks the exact CPU
+  model of the compute host against the requested compute instance model. It
+  will only pick compute hosts that have the <literal>cpu_model</literal>
+  requested by the instance model, and if the compute host does not have that
+  <literal>cpu_model</literal>, the ComputeCapabilitiesFilter moves on to find
+  another compute host that matches, if possible. Selecting an outdated vCPU
```

Comment:
* You left out the part that libvirt can actually emulate previous-generation CPUs, just Nova doesn't expose that properly in its ComputeCapabilitiesFilter. If that issue is not mentioned, the  new scheduler filter looks a bit out of context.
* Should this word really be "outdated" - what about "unavailable"?